### DISCUSSION TITLE
So.. Will we need to pay Colab now?

### DISCUSSION THREAD

**Discussion by cutecoelho** [TheLastBen/fast-stable-diffusion#2106] [Q&A]
I'm been disconnected from Colab everytime.. this is annoying. So its over the freedom from free users?

If the answer is yes, each one is the best option?

Colab, RunPod or Paperspace?

**Comment by TheLastBen** [comment]
Avoid colab pro, use Runpod or Paperspace, they offer a wide range of GPUs at adequate prices.

**Reply by meunumerotim333** [reply]
Considering I was using Colab all day.. I've calculated the prices.. Will be better buy a new GPU. I'll continue using DreamBooth Colab to train. Would be very cool if you keep updated buddy . At least DB still working

**Reply by ClaraSanders** [reply]
What I don't like about Runpod is the lack of fixed plans. I hate to have to pay each day for something that I use like four hours a day.

**Reply by Pagangawd** [reply]
So you've stopped supporting colab?

**Reply by Nojarosann** [reply]
i think he'll just not give it as much attention considering the entire reason people have been using colab in the first place is because it was free

**Reply by cutecoelho** [reply]
Please continue Supporting DreamBooth on Google Colab. I'm still using a lot there. I train there and use on PC.

**Reply by TheLastBen** [reply]
the notebooks will be maintained, don't worry

**Comment by juone06** [comment]
i face same prob..after run stable diffussion, colab disconnect

**Reply by ClaraSanders** [reply]
This is much worse than you think. I have been unable to connect to Google Colab for three days. Not only SD, but ANY Colab repo. It is like one of my free accounts has been permabanned without any email or advice. The best part is that thanks to Drive running like shit I have problems using Last Ben's even with my paid account. Right now I don't know if there is a new issue or if it just doesn't work because the repo can't connect to Drive.

**Reply by TheLastBen** [reply]
go to runpod, it's less hassle

**Reply by ClaraSanders** [reply]
Yeah, but I don't like having to pay each day for a service, I want a fixed plan, you know? Like 5 hours per day XXX money.

**Reply by TheLastBen** [reply]
5 hours per day is 1$ for double the speed of a T4
RTX 3070 is 0.2$/h

**Reply by ClaraSanders** [reply]
Since you are a pro at this, what kind of machine I would need to hire to run these new GPT models from MPT like Storywriter? Is this even possible from a remote machine?

**Reply by TheLastBen** [reply]
I'm not familiar with those models

**Reply by MasterDenis** [reply]
> 5 hours per day is 1$ for double the speed of a T4 RTX 3070 is 0.2$/h

few questions regarding runpod.
for casual use which is the most recommended setup for the cheapest expenses but decent speed (mostly cheapest because I am a student)
I also saw some competitive options on vast.ai things like 3090 for 0.2/h but unsure.

**Reply by vonvanzu** [reply]
I have a complaint using runpod in regard to "intellectual properties rights" they ask you to accept before sign up to start to use their services. They say something like the 'sites and the marketplace offerings are their property and all software, machines etc, are theirs too'. So basically they are telling that everything there in the site included any images generate in the machines are theirs??? 

* please read the TERMS OF SERVICES and let me know. 

Best Regards

**Reply by TheLastBen** [reply]
@vonvanzu what you do in the VM is private to you, no one has access to it but you, so no one else can claim ownership over it

**Comment by MasterDenis** [comment]
Okay so do we need to pay now for running it casually? I generally do not do anything other than generating some pictures so paying for sub may not be the most ideal use case solution.
What is paperspace and do they have a free tier for more casual use?
Also for runpod how much would it cost?
I am willing to pay but I am a student and have no much need for hourly use.

**Reply by TheLastBen** [reply]
as I said before, a 3070 which is double the speed of the free colab GPU is at 0.2$/h

**Reply by MasterDenis** [reply]
> as I said before, a 3070 which is double the speed of the free colab GPU is at 0.2$/h

yeah saw, also would there be much benefit in getting one of the A4000/A4500 instances?

also paperspace seems to have phone verification issues rn, I will try later.

**Comment by Headupdef** [comment]
paperspace  has been sucking the last few days with a slew of maintenance.
